quakemigrate/plot/event.py

```diff
@@ -100,13 +100,31 @@ def event_summary(run, event, marginalised_coa_map, lut, xy_files=None):
     _plot_text_summary(text, lut, event)
 
     # Deal with repeating labels in waveform gather legend; combine labels for
-    # ["N", "1"], ["E", "2"]
+    # e.g. ["N", "1"], ["E", "2"]; and same for P (e.g. ["Z", "1"]) -- based on
+    # components supplied for each phase in onset.channel_maps
+    p_str, s_str_1, s_str_2 = util.get_phase_component_strings(
+        event.onset_data.channel_maps
+    )
```

Comment:
Hmm, just wondering if there's a technical distinction between the onset channel maps and the explicit number of channels that go into computing the onsets.

Also, want to test what happens if we run with a single phase.